client/extensions/woocommerce/app/settings/payments/payment-method-stripe.js

```diff
@@ -69,126 +107,265 @@ class PaymentMethodStripe extends Component {
 		this.props.onEditField( e.target.name, e.target.value );
 	}
 
-	onToggleTestMode = ( mode ) => {
-		const testmode = mode === 'test' ? 'yes' : 'no';
-		// return curried function
-		return () => {
-			this.props.onEditField( 'testmode', testmode );
-		};
+	hasNonTrivialStringValue = ( prop ) => {
+		if ( ! ( 'value' in prop ) ) {
+			return false;
+		}
+		if ( ! isString( prop.value ) ) {
+			return false;
```

Comment:
Should this be an error of some kind?

Reply:
This might not actually be possible to occur after all. Now that I think about it, this might be preventative code left over from an earlier design. I'll investigate.